packages/block-library/src/rss/edit.js

```diff
@@ -117,7 +117,14 @@ export default function RSSEdit( { attributes, setAttributes } ) {
 			isActive: blockLayout === 'grid',
 		},
 	];
-
+	const serverSideAttributes = {
```

Comment:
Should we add a comment here to explain why the styles needs to be reset?

Reply:
Sure, included the comment in the recent commit.